src/engine/objects/resolvers/visitors.rs

```diff
@@ -929,14 +1010,23 @@ where
                 props_type_name,
                 partition_key_opt,
             )
-            .await
-            .and_then(|rels| {
-                if let Some(handlers) = context.event_handlers().after_rel_create(&rel_label) {
-                    handlers.iter().try_fold(rels, |v, f| f(v))
-                } else {
-                    Ok(rels)
-                }
-            })
+            .await?;
+        if let Some(handlers) = context.event_handlers().after_rel_create(&rel_label) {
+            for f in handlers.iter() {
+                rels = f(
+                    rels,
+                    EventFacade::new(
+                        // TODO: where do we get the rel_name ?
```

Comment:
The `RelQueryVar` struct has a function to obtain it, so it'd likely be `rel_var.label()`.

Reply:
Fixed.